mango-python/bdgenomics/mango/variants.py

```diff
@@ -76,3 +83,165 @@ def viewPileup(self, contig, start, end, reference = 'hg19', label = "Variants",
             locus="%s:%i-%i" % (contig, start, end)
             return pileup.PileupViewer(locus=locus, reference=reference, tracks=tracks)
 
+
+class VariantsPerSampleDistribution(object):
+    """ VariantsPerSampleDistribution class.
+    VariantsPerSampleDistribution computes a distribution of the count of variants per sample.
+    """
+    def __init__(self,ss,genotypeRDD):
+        """
+        Initializes a VariantsPerSampleDistribution class.
+        Computes the counts of variants per sample from a genotypeRDD.
+
+        :param ss: SparkContext
+        :param genotypeRDD: bdgenomics genotypeRDD
+        """
+        df =  genotypeRDD.toDF()
+        data =  df.where( (df.alleles[0] == u'ALT') | (df.alleles[1] == u'ALT')).groupBy("sampleid").count().collect()
+        self.counts = [ sample.asDict()['count'] for sample in data ]
+
+    def plot(self, testMode = False, **kwargs):
+        """
+        Plots final distribution values and returns the plotted distribution as a list
+
+        Returns:
+          matplotlib axis to plot and computed data
+        """
+
+        if(not testMode):
+            figsize = kwargs.get('figsize',(10,5))
+            fig, ax = plt.subplots(figsize=figsize)
+            bins = kwargs.get('bins',100)
+            hist = ax.hist(self.counts, bins=bins)
+            return ax, self.counts
+        else:
+            return None, self.counts
+
+
+class HetHomRatioDistribution(object):
```

Comment:
I still think this is a weird name for a class, considering "HetHom" is an abbreviation. Because this is an exposed API, we need to think about what makes sense to the user.

Reply:
HeterozyoteToHomozyogteRatioDistribution ?
I can't find any other shorter name for this statistic.
Maybe:
GenotypeRatioDistribution 
thoughts it is less correct